src/assets/plugins/app.header.json

```diff
@@ -47,15 +47,6 @@
             "actions": {
               "click": "app.actions.about"
             }
-          },
-          {
-            "id": "app.header.logout",
```

Comment:
Can you try to make it disabled? This could be a good example for people how to build such menu.

Reply:
per conversation, we cannot disable it to remove it